src/search/components/SearchForm.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {
+  onSearch: PropTypes.func.isRequired,
+};
+
+class SearchForm extends React.Component {
+  constructor() {
+    super();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
```

Comment:
Looking into it, it seems that projects like redux removed their use of < stage 2 stuff to try and not put people off using that project because they might think it's a requirement.